docs/docs/quickstart.md

```diff
@@ -185,7 +187,7 @@ With this creation rule, datapoints added to the `sensor1` timeseries will be gr
 
 
 ## Filtering
-RedisTimeSeries allows to filter by value, timestamp and by labels:
+Redis Time Series allows to filter by value, timestamp and by labels:
```

Comment:
Again - suggest removing "Redis Time Series". Consider rewriting as "You can filter data by..."

Reply:
Thanks. Changing to "You can filter yor time series by value, timestamp and labels"